/wheel_calc:
  ros__parameters:
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    b: 0.84 # Wheel base
    r: 0.143 # Wheel radius
    t: 0.84 # wheel track
    twist_topic: /wheel/twist
    joint_state_topic: /wheel/wheel_odom
    joint_state_topic: /wheel/joint_states
    use_sim_time: false

